.gitignore

```diff
@@ -144,3 +144,17 @@ config/cable.yml
 .yarn/build-state.yml
 .pnp.*
 package-lock.json
+
+# PortaFly
+portafly/node_modules
+
+## env
+!/portafly/.env
```

Comment:
Is this `!` correct? Is this file to be ignored?

Reply:
That's correct, that file should **not** be ignored